src/proto/h1/role.rs

```diff
@@ -308,40 +307,40 @@ impl Http1Transaction for Server {
                             encoder = Encoder::length(known_len);
 
                             #[cfg(debug_assertions)]
-                            {
```

Comment:
The diff makes this hard to tell, but did anything changes besides indentation? Could we revert the indent, so as to reduce rightward shift?